example/ConfidenceBand.purs

```diff
@@ -0,0 +1,242 @@
+        width = Just 1.0
+        }
+      },
+    splitLine = Just $ AxisSplitLine axisSplitLineDefault {
+      show = Just false
+      },
+    splitArea = Just $ AxisSplitArea axisSplitAreaDefault {
+      areaStyle = Just $ AreaStyle areaStyleDefault { 
+                color = Just $ SimpleColor "rgba(255,255,255,1)"           
+        }
+      },
+    axisLabel = Just $ AxisLabel axisLabelDefault {
+      formatter = Just $ NumberFormatFunc (
+        numeralFormatterWithValMnplt data_base 1.0 "0.00"),
+      textStyle = Just $ TextStyle textStyleDefault {
+       fontFamily = Just "Palatino, Georgia, serif"
+       }
+      }
+    },
+
+  series = Just $ Just <$> [
+    LineSeries {
+      common: universalSeriesDefault {
+        name = Just "value",
+        itemStyle = Just $ ItemStyle itemStyleDefault {
+          normal = Just $ IStyle istyleDefault {
+            color = Just $ SimpleColor "rgba(0,119,215,1)"
+            }     
+          }
+        },
+      lineSeries: lineSeriesDefault {
+        symbol = Just $ NoSymbol,
+        smooth = Just false,
+        "data" = Just $ simpleData <$> ((\n->n-data_base) <$> data_val)
+        }
+      },
+    LineSeries {
+      common: universalSeriesDefault {
+        name = Just "lower boundary",
+        itemStyle = Just $ ItemStyle itemStyleDefault {
+          normal = Just $ IStyle istyleDefault {
+            lineStyle = Just $ LineStyle lineStyleDefault {
+              width= Just 0.0
+              },
+            areaStyle = Just $ AreaStyle areaStyleDefault { 
+                color = Just $ SimpleColor "rgba(255,255,255,1)"           
+              }            }     
+          }
+        },
+      lineSeries: lineSeriesDefault {
+        symbol = Just $ NoSymbol,
+        smooth = Just false,
+        "data" = Just $ simpleData <$> ((\n->n-data_base) <$> data_l)
+        }
+      },
+    LineSeries {
+      common: universalSeriesDefault {
+        name = Just "upper boundary",
+        itemStyle = Just $ ItemStyle itemStyleDefault {
+          normal = Just $ IStyle istyleDefault {
+            lineStyle = Just $ LineStyle lineStyleDefault {
+              width= Just 0.0
+              },
+            areaStyle = Just $ AreaStyle areaStyleDefault { 
+                color = Just $ SimpleColor "#ebf6ff"           
+              }
+            }     
+          }
+        },
+      lineSeries: lineSeriesDefault {
+        symbol = Just $ NoSymbol,
+        smooth = Just false,
+        "data" = Just $ simpleData <$>  ((\n->n-data_base) <$> data_u)
+        }
+      }
+    ]
+  }
+
+
+
+confidenceBand id = do
+  mbEl <- getElementById id
+  case mbEl of
+    Nothing -> print "incorrect id in confidence-band"
+    Just el -> do
+      chart <- init Nothing el
+      chart' <- setOption options true chart
+      pure unit
+
+data_date = ["2012-08-28 ", "2012-08-29 ", "2012-08-30 ", "2012-08-31 ", "2012-09-01 ", "2012-09-02", "2012-09-03 ", 
```

Comment:
PureScript use camelCase. This is about indentation too :) Something like 
```purescript
dataDate = 
  [ "2012-08-28"
  , "2012-08-29" 
  ...
  ]
```
Oh, and could you please add type annotation to all type level definitions?